scripts/pg2sphinx.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -euo pipefail
 
+docker_is_logged_in() {
+    jq -e --arg url "${DOCKER_REGISTRY}" '.auths | has($url)' ~/.docker/config.json > /dev/null
+}
+
+# check if we are already logged in
+if ! docker_is_logged_in; then
```

Comment:
ok, no [ brackets neccesar as the function returns a boolean

Reply:
returns only the exit code:

- ` 0 ` for logged in 

- `<> 0` for any other state or issue